public/app/plugins/datasource/loki/querybuilder/state.ts

```diff
@@ -42,7 +42,7 @@ export function getQueryWithDefaults(query: LokiQuery): LokiQuery {
   }
 
   if (query.expr == null) {
-    result = { ...result, expr: '' };
+    result = { ...result, expr: '{} |= ``' };
```

Comment:
It seems that this is breaking the editor. See video:

https://user-images.githubusercontent.com/1069378/203364439-1139dc8b-f89f-4859-9fd2-4f9217387437.mov

I would leave it as it previously was. If we want to change the default, we can do it in another PR.